string[] joint_names
float64[] positions
duration time_from_start